SyncSettings.py

```diff
@@ -1,12 +1,17 @@
 # -*- coding: utf-8 -*-
 
-import sublime
-import sys
 import os
+import sys
+
+import sublime
+
+from .sync_settings.libs import path, settings
 
-f = os.path.join(os.path.expanduser('~'), '.sync_settings')
-if not os.path.isdir(f):
-    os.mkdir(f)
+default_file_path = path.join(os.path.expanduser('~'), '.sync_settings', 'sync.json')
+file_path = settings.get("config_location") or default_file_path
+
+if not os.path.isdir(file_path):
+    os.mkdir(file_path)
```

Comment:
what happen if sublime text is not allowed to perform any action over the configured path? Try to have a fallback for those escenarios.

Reply:
Attempted to include this logic.